src/sentry/api/event_search.py

```diff
@@ -438,6 +442,9 @@ def to_query_string(self) -> str:
             return str(self.value)
 
     def is_wildcard(self) -> bool:
+        # If we're using the raw value only it'll never be a wildcard
+        if self.use_raw_value:
+            return False
```

Comment:
do you need this here if we're already skipping the `is_wildcard` check here? https://github.com/getsentry/sentry/pull/88249/files#diff-0b239bf9fccf786d1ac7bd5498e7133b828157098100ba72ceae29b2254ca915R423-R424

Reply:
Good q, There are checks elsewhere in the code for `SearchValue.is_wildcard` that we need to be `False` so we need it in both locations.

It's a bit weird cause it feels like this check here makes 423 unnecessary too, but since the raw_value is a string we need the check in both methods